ngrok/src/session.rs

```diff
@@ -463,6 +544,18 @@ impl SessionBuilder {
         self
     }
 
+    /// Configures the session to connect to ngrok through an outbound
+    /// HTTP or SOCKS5 proxy. This parameter is ignored if you override the connector
+    /// with [SessionBuilder::connector].
+    ///
+    /// See the [proxy url paramter in the ngrok docs] for additional details.
```

Comment:
speling nit: `paramter`

Reply:
:grin: 

guess I'll need to fix that in Go as well, since this was copied

Reply:
lol, another one of _those_ problems